### ISSUE TITLE
Removes item in players mouse slot in inventory when removing a potion effect

### ISSUE THREAD

**Issue by Rodney-RWR** [SkriptLang/Skript#5237]
### Skript/Server Version

```markdown
[22:19:10 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[22:19:10 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[22:19:10 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[22:19:10 INFO]: [Skript] Server Version: git-Paper-387 (MC: 1.18.2)
[22:19:10 INFO]: [Skript] Skript Version: 2.6.3
[22:19:10 INFO]: [Skript] Installed Skript Addons: 
[22:19:10 INFO]: [Skript]  - skript-placeholders v1.5.2 (https://github.com/APickledWalrus/skript-placeholders)
[22:19:10 INFO]: [Skript]  - DiSky v4.1.1
[22:19:10 INFO]: [Skript]  - skRayFall v1.9.27 (https://sk.rayfall.net/)
[22:19:10 INFO]: [Skript]  - SkQuery v4.1.6
[22:19:10 INFO]: [Skript]  - SkBee v2.4.0 (https://github.com/ShaneBeee/SkBee)
[22:19:10 INFO]: [Skript]  - Ersatz v1.0
[22:19:10 INFO]: [Skript] Installed dependencies: 
[22:19:10 INFO]: [Skript]  - Vault v1.7.3-b131
[22:19:10 INFO]: [Skript]  - WorldGuard v7.0.5+3827266
```

### Bug Description

When removing a potion effect the item the player has in their cursor in the inventory is removed. Aswell as removing the potion effect.

### Expected Behavior

Just remove the potion effect, don't delete item from cursor.

### Steps to Reproduce

apply a potion effect, click and item in your inventory and wait for a usage of `remove <effectname> from player` 

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by Trirha** [comment]
if you want it to remove the item from the main hand that the player is holding do

`   set player's held item to air`

**Comment by Rodney-RWR** [comment]
> if you want it to remove the item from the main hand that the player is holding do
> 
> ` set player's held item to air`

1. That is clearly the wrong answer
2. This is a bug report involving removing effects

**Comment by Trirha** [comment]
> > if you want it to remove the item from the main hand that the player is holding do
> > ` set player's held item to air`
> 
> 1. That is clearly the wrong answer
> 2. This is a bug report involving removing effects

welp i tried \_( )_/

**Comment by bluelhf** [comment]
This does not sound like a Skript issue. Is it possible this is caused by Spigot, or even Minecraft itself?

**Comment by DelayedGaming** [comment]
Can't reproduce bug on Paper 1.19.3-325 with Skript 2.6.4.

**Comment by Rodney-RWR** [comment]
> Can't reproduce bug on Paper 1.19.3-325 with Skript 2.6.4.

I will try with 2.6.4 but im on paper 1.18.2

**Comment by TheLimeGlass** [comment]
2.6.4 supports 1.13-latest

**Comment by DelayedGaming** [comment]
Fixed in #5476

**Comment by TheLimeGlass** [comment]
Fixed in 2.7.0-beta2
